src/styles/header.scss

```diff
@@ -0,0 +1,69 @@
+.header {
+  font-family: "Roboto", sans-serif;
```

Comment:
You can specify the font family for the HTML selector to get it inherited everywhere